lib/commands/start.js

```diff
@@ -41,7 +48,7 @@ class StartCommand extends Command {
             ].join('\n'), 'yellow');
         }
 
-        await this.runCommand(DoctorCommand, {categories: ['start'], ...argv, quiet: true});
+        await this.runCommand(DoctorCommand, {categories: ['start'], ...argv, quiet: true, skipInstanceCheck: true});
```

Comment:
If we add support for named update, I think we can get rid of this `skipInstanceCheck` flag, since it will be true in all places where the `doctor` command is run. Plus, `start`, `stop`, and `update` will have similar command structures!